rust/connlib/tunnel/src/dns.rs

```diff
@@ -408,7 +390,7 @@ where
 
     if records.is_empty() {
         tracing::debug!(
-            "No {qtype} records for {}, returning NXDOMAIN",
```

Comment:
We no longer need this really, since if we get here it can only be `A` or `AAAA` and the only case we get here is IP exhaustion, so `NXDOMAIN` seems like a good idea to make the querying application stop.

This is just an extremely weird edge-case that I think it won't matter what we do here.

Reply:
Can we just not have this code snippet and return empty records?